### PR TITLE
Feature/cloudwatch target

### LINKED ISSUE #145: Cloudwatch datasources not supported due to Target class restrictions
There is no support for Cloudwatch since the Target class has restrictions. If you remove them, you'll be able to make Cloudwatch graphs.

### PR DESCRIPTION
**Pull request by Puneeth-n**
closes #145

### CODE REVIEW

**Comment by Puneeth-n** [comment]
Hi @JamesGibo  @dholbach Can you please review it?

File: grafanalib/tests/test_cloudwatch.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+"""Tests for Cloudwatch Datasource"""
+
+import grafanalib.core as G
+import grafanalib.cloudwatch as C
+from grafanalib import _gen
+
+import sys
+if sys.version_info[0] < 3:
```
Comment (JamesGibo) [inline grafanalib/tests/test_cloudwatch.py:8]: We probably don't need this as there is no support for py2 now in grafanalib

**Review by JamesGibo — CHANGES_REQUESTED** [review]
Thanks for your PR it is much appreciated and a feature than many users have requested. 
I don't use cloudwatch myself so have not tested the code but what you have done seems to cover the features defined in the Grafana docs.

Please could you add a note to the changelog about what you have added?

**Review by JamesGibo — APPROVED** [review]
LGTM

**Pull request merged**